README.md

```diff
@@ -1,7 +1,10 @@
 # fonto-benchmark-runner [![Build Status](https://travis-ci.org/FontoXML/fonto-benchmark-runner.svg?branch=master)](https://travis-ci.org/FontoXML/fonto-benchmark-runner)
+
 This tool can measure the performance of a function and compare the performances of different functions in both NodeJS and in the browser. It takes Typescript files as input.
 
 How to use:
 
-- Create files in a performance folder in your project with names ending with .benchmark.ts
-- Run `npx fonto-benchmark-runner` to run your performance tests in node.
+-   Create files in a performance folder in your project with names ending with .benchmark.ts
+-   Run `npx fonto-benchmark-runner` to run your performance tests in node.
+-   Add `--excel` to the command in order to log the results in compliance with csv files. If you need to write the results
```

Comment:
I'd prefer if we do not specify excel. The format we allow to output is CSV, this is a format which excel can read we are not outputting the excel format (xlsx).
So maybe something like `--output-csv`.

I'd also like it if we can rename all other variable names and such to no longer mention excel but indicate we are trying to output in the CSV format.